docs/design/hierarchical-queue-on-capacity-plugin.md

```diff
@@ -0,0 +1,133 @@
+# Hierarchical Queue on Capacity Plugin
+
+## Motivation
+
+In the context of cloud-native AI task scheduling, fair scheduling and resource utilization are two critical concerns for users. The Volcano community has already developed the elastic queue management plugin, known as capacity, to support fine-grained resource borrowing/lending and queue management, thereby improving resource utilization. However, in real-world scenarios, queues are often hierarchical, corresponding to the hierarchical organizational structure of company teams. To better align with practical queue usage scenarios and further enhance resource utilization for AI task scheduling, Volcano needs to build upon the existing capacity plugin and introduce hierarchical queue management capabilities.
+
+## In Scope
+
+- Enhancing the elastic queue management capabilities of the capacity plugin to support hierarchical queues.
+
+## Out of Scope
+
+- Extending the hierarchical queues of any other plugins.
```

Comment:
If the capacity plugin is not enabled, will the hierarchical queue still be valid? For example, is it possible to submit tasks to a subqueue?

Reply:
If the Capacity plugin is not enabled, the hierarchical queue feature will not take effect; however, it's important to note that the DRF plugin also supports hierarchical queues, and users can utilize the hierarchical queue functionality provided by other plugins.
When a task is submitted to a child queue, the Capacity plugin does not perform any validation; it primarily depends on the validation of other plugins.

Reply:
Can this vcjob be submitted normally?

Reply:
yes